Tools/Plotting/DalitzPlot.cpp

```diff
@@ -65,9 +65,9 @@ void DalitzPlot::fill(const std::vector<ComPWA::Event> &data, bool normalize,
   _plotHistograms.push_back(std::move(hist));
 }
 
-void DalitzPlot::fill(const std::vector<ComPWA::Event> &data,
-                      FunctionTreeIntensity &intens, bool normalize,
-                      std::string name, std::string title, Color_t color) {
+void DalitzPlot::fill(const std::vector<ComPWA::Event> &data, Intensity &intens,
+                      bool normalize, std::string name, std::string title,
```

Comment:
Why no `const std::string&`?

Reply:
For simply opjects like `std::string` or `double` I doubt that there is any difference after compiler optimizations. For larger objects like `const std::vector<ComPWA::Event> &data` one should certainly pass by reference.

Reply:
Yeah, but just a convention thing
https://github.com/ComPWA/ComPWA/blob/master/CONTRIBUTING.md#const-correctness

Reply:
For `double` a reference would actually be slower btw because of dereferencing, and the size of the pointer is also the same